### PR TITLE
deb: add workaround for setuptools-scm issue

### PR DESCRIPTION
**Pull request by dtrifiro**
fix broken deb build due to upstream setuptools-scm issue (see comment below)

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -26,15 +26,16 @@ def __init__(
...
+    def get_pkg_build_cmd(self, tag_or_branch: str) -> List[str]:
         return [
             "bash",
             "-c",
             " && ".join(
                 [
-                    "pip install './dvc[all]'",
-                    "pip install -r dvc/scripts/build-requirements.txt",
-                    f"python dvc/scripts/build.py {self.pkg}",
+                    f"git clone --branch '{tag_or_branch}' https://github.com/iterative/dvc /dvc",
```
Comment (efiop) [inline utils.py:35]: One problem with this approach is that we are no longer able to debug locally by building our own local dvc copy

Comment (efiop) [reply utils.py:35]: Maybe we can chmod/chown the clone instead?

Comment (dtrifiro) [reply utils.py]: Found a workaround: `git config --global --add safe.directory '*'`

File: utils.py [hunk 2]
```diff
@@ -32,6 +32,7 @@ def get_pkg_build_cmd(self) -> List[str]:
             "-c",
             " && ".join(
                 [
+                    "git config --global --add safe.directory '*'",
```
[the remark this answers is not in the document]
Comment (dtrifiro) [reply utils.py:35]: It's actually in the commit message, would you also like a comment?

**Review by efiop — APPROVED** [review]
Thank you! 🙏

**Pull request merged**